docs/tutorials/contribution_workflow.md

```diff
@@ -1,16 +1,16 @@
 
 ## Contribution Workflow
 
-#### 1. Example Contribution Workflow using DevContainer:
+#### 1. Example Contribution Workflow using the R Dev Container:
 -  To start working in R we will click on `R:(not attach)` option which is in the bottom right of our R-dev codespace. It will open R terminal for us.
```

Comment:
```suggestion
-  To start working in R we will click on `R:(not attach)` which is in the bottom right of the VSCode window. This will open an R terminal for us.
```